src/packages/PkgCompat.jl

```diff
@@ -159,7 +159,7 @@ end
 # (⛔️ Internal API)
 _get_registries() = map(_get_registry_paths()) do r
```

Comment:
Here, the entire function `_get_registries` can be cleaned up, because `RegistryInstances` means that we no longer need to use the `@static if` with `isdefined` to find out where the API is. It should look something like:

```julia
_get_registries() = map(_get_registry_paths()) do r
	RegistryInstances.RegistryInstance(r)
end
```

Same for `_get_registry_paths`

Reply:
We can now also update the comments with ⛔️

Reply:
Good point! It looks much cleaner now!

```julia
# (✅ "Public" API using RegistryInstances)
"Return paths to all installed registries."
_get_registry_paths() = [s.path for s in RegistryInstances.reachable_registries()]

# (✅ "Public" API using RegistryInstances)
_get_registries() = map(_get_registry_paths()) do r
	RegistryInstances.RegistryInstance(r)
end

# (✅ "Public" API)
"Contains all registries as `Pkg.Types.Registry` structs."
const _parsed_registries = Ref(_get_registries())

# (✅ "Public" API)
"Re-parse the installed registries from disk."
function refresh_registry_cache()
	_parsed_registries[] = _get_registries()
end
```

Reply:
Awesome!